patches/chrome-browser-shell_integration_win.cc.patch

```diff
@@ -56,9 +56,14 @@ index 9b05880c7b94700e2cbfdddcd8e0b95cd11f7c4e..f6b4919378cc72bffe1e95d261566364
 +                       app_cmd.end());
 +  }
 +  return std::string();
-+}
-+
-+bool IsIEDefaultBrowser() {
+ }
+ 
+ bool IsIEDefaultBrowser() {
+-  base::string16 app_cmd;
```

Comment:
this doesn't seem right, why are we removing these lines?

Reply:
This logic is now encapsulated in GetHttpProtocolUserChoiceProgId()

Reply:
Looks like we are pulling this functionality from c80, though. It's not in c79 yet.

Reply:
https://github.com/brave/brave-core/commit/d804c06d7f30809716a1c37e690e550df84a00a5#diff-1bf5d30576ed6b563ab222e211d62a4cR61